monai/data/image_reader.py

```diff
@@ -121,6 +121,10 @@ class ITKReader(ImageReader):
     def __init__(self, **kwargs):
         super().__init__()
         self.kwargs = kwargs
+        if itk.Version.GetITKVersion() < "5.2.1":
```

Comment:
This should be 5.2.0, as Hans' fix should make it into 5.2.0.

Reply:
perhaps should use numerical comparisons `int(itk.Version.GetITKMajorVersion()) == 5 and int(itk.Version.GetITKMinorVersion()) <= 2 and int(itk.Version.GetITKBuildVersion()) <= 0 `

Reply:
@wyli,  yes!  my code was just psuedocode.  I don't think the above code is valid.

Reply:
Thanks for your suggestions.
Updated in the latest commit.

Thanks.